src/plugins/intel_cpu/src/node.h

```diff
@@ -739,10 +745,14 @@ class Node {
         IE_THROW(NotImplemented) << "[DS] prapareParams not implemented for node with type " << NameFromType(getType());
     }
 
-    MultiCachePtr getRuntimeCache() const {
+    MultiCachePtr getRuntimeCache() {
```

Comment:
Could you please elaborate on why it was decided to remove the `const` qualifier?

Reply:
Sorry, will fix that mistake